Quadriga/src/main/java/edu/asu/spring/quadriga/service/impl/workbench/ModifyProjectManager.java

```diff
@@ -98,32 +109,34 @@ public void updateProject(String projID, String projName,String projDesc,String
 	
 	/**
 	 * This method deletes a project into the database.
+	 * 
 	 * @param project
-	 * @return String - error message blank on success and contains error on failure.
+	 * @return String - error message blank on success and contains error on
+	 *         failure.
 	 * @throws QuadrigaStorageException
 	 * @author Karthik Jayaraman, Julia Damerow
 	 */
 	@Override
 	@Transactional
-	public void deleteProjectRequest(List<String> projectIdList) throws QuadrigaStorageException
-	{
-	    for(String projectId : projectIdList) {
-            ProjectDTO  project = projectDao.getProjectDTO(projectId);
-            //retrieve all the foreign key tables associated with them
-            //delete the project workspace DTO
-            deleteProjectWorkspaceMapping(project);
-            deleteProjectDictionaryMapping(project);
-            deleteProjectConceptCollectionMapping(project);
-            deleteProjectCollaboratorMapping(project);
-            deleteProjectEditorMapping(project);
-            
-            projectDao.updateDTO(project);    
-            projectDao.deleteDTO(project);
-        }
+	public void deleteProjectRequest(List<String> projectIdList) throws QuadrigaStorageException {
+		for (String projectId : projectIdList) {
+			ProjectDTO project = projectDao.getProjectDTO(projectId);
+			// retrieve all the foreign key tables associated with them
+			// delete the project workspace DTO
+			deleteProjectWorkspaceMapping(project);
+			deleteProjectDictionaryMapping(project);
+			deleteProjectConceptCollectionMapping(project);
+			deleteProjectCollaboratorMapping(project);
+			deleteProjectEditorMapping(project);
+
+			projectDao.updateDTO(project);
+			projectDao.deleteDTO(project);
+		}

```

Comment:
make sure to use correct indentation according to style guide:
http://diging.github.io/course-book/2015/07/10/code-style.html